src/controllers/comment.controller.js

```diff
@@ -0,0 +1,56 @@
+import commentService from '../services/comment.service';
+import response from '../helpers/response.helper';
+import Paginate from '../helpers/paginate.helper';
+
+
+/**
+ * This class will controll all action related to the
+ * comment
+ */
+class comment {
+  /**
+     * This method will help
+     * to comment to the trip request
+     * @param {Object} req request data
+     * @param {Object} res response data
+     * @returns { Object} return a user message
+     */
+  static async createTripRequestComment(req, res) {
+    const Id = req.params.tripRequestID;
+    const result = await commentService.getComments(req, Id);
+    if (result) return response.successMessage(res, 'comment created successfuly', 201);
+    return response.errorMessage(res, 'Comment already exist', 409);
+  }
+
+  /**
+   *
+   * This method will help to view all
+   * comment tha has been commented on
+   * a spacifi trip request
+   * @param {Object} req user request data
+   * @param {Object} res user response data
+   * @returns { Object} return a user message
+   */
+  static async getAllComments(req, res) {
+    const Id = req.params.tripRequestID;
+    const { page } = req.query;
+    const limitNumber = 10;
+    const offset = Paginate(page, limitNumber);
+    const data = await commentService.getAllCommets(Id, 'trip request', limitNumber, offset);
```

Comment:
@shemaeric1234 In order to make this getAllComments function more generic, you have to pass all parameters (i.e. subjectID, subjectType, etc....) instead of hardcoding them.